src/OpenColorIO/ColorSpace.cpp

```diff
@@ -147,6 +148,16 @@ const char * ColorSpace::getAlias(size_t idx) const noexcept
     return "";
 }
 
+bool ColorSpace::hasAlias(const char * alias) const noexcept
+{
+    for (size_t idx = 0; idx < getImpl()->m_aliases.size(); ++idx) {
```

Comment:
Very minor: The coding style for the project moves the opening bracket "{" to its own line.